base/applications/rapps/misc.cpp

```diff
@@ -79,6 +79,25 @@ ShowPopupMenuEx(HWND hwnd, HWND hwndOwner, UINT MenuID, UINT DefaultItem)
     }
 }
 
+HTREEITEM
+TreeView_Walk(HWND hTree, TVWALKCALLBACK Callback, LPARAM Cookie, BOOL Children, HTREEITEM hRoot)
```

Comment:
I would suggest you rename that `LPARAM Cookie` to `PVOID Context`, which is a more standard naming ("Cookie" is more like for some "magic" number). Also, using `LPARAM` is quite baroque (it's not as if you were writing a window-message handler ^^)

Reply:
Take a look at [EnumWindows](https://learn.microsoft.com/en-us/windows/win32/api/winuser/nf-winuser-enumwindows) or sort callbacks in comctl32, they all use LPARAM.

Reply:
Yes, because they come from old-school win32/win16 where LPARAM/WPARAM was used.
Anyway, please change "Cookie" to some other name ("Context", "Param")

Reply:
Already did